Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -144,54 +144,56 @@ static void APIENTRY ClearDepthf(GLfloat depthval)
 
 static void InitDriverInfo()
 {
+  const std::string_view svendor(g_ogl_config.gl_vendor);
+  const std::string_view srenderer(g_ogl_config.gl_renderer);
+  const std::string_view sversion(g_ogl_config.gl_version);
   DriverDetails::Vendor vendor = DriverDetails::VENDOR_UNKNOWN;
   DriverDetails::Driver driver = DriverDetails::DRIVER_UNKNOWN;
   DriverDetails::Family family = DriverDetails::Family::UNKNOWN;
   double version = 0.0;
 
   // Get the vendor first
-  if (!strcmp(g_ogl_config.gl_vendor, "NVIDIA Corporation"))
+  if (svendor == "NVIDIA Corporation")
   {
-    if (!strcmp(g_ogl_config.gl_renderer, "NVIDIA Tegra"))
+    if (srenderer != "NVIDIA Tegra")
     {
-      vendor = DriverDetails::VENDOR_TEGRA;
+      vendor = DriverDetails::VENDOR_NVIDIA;
     }
     else
     {
-      vendor = DriverDetails::VENDOR_NVIDIA;
+      vendor = DriverDetails::VENDOR_TEGRA;
     }
   }
-  else if (!strcmp(g_ogl_config.gl_vendor, "ATI Technologies Inc.") ||
-           !strcmp(g_ogl_config.gl_vendor, "Advanced Micro Devices, Inc."))
+  else if (svendor == "ATI Technologies Inc." || svendor == "Advanced Micro Devices, Inc.")
   {
     vendor = DriverDetails::VENDOR_ATI;
   }
-  else if (strstr(g_ogl_config.gl_version, "Mesa") != nullptr)
+  else if (std::string::npos != sversion.find("Mesa"))
```

Comment:
Our convention is to structure things like this as: `sversion.find("Mesa") != std::string::npos`. There's no benefit gained through the other method aside from looking slightly more obtuse to a reader. This is also more consistent with existing code. Ditto for other occurrences in the change.

Reply:
The flipped version is how the code was originally before my changes. I'll unflip it anyway.

Reply:
Ah, OK, I didn't realize it was a revert. Thanks!